sonar-project.properties

```diff
@@ -0,0 +1,32 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
```

Comment:
Curious, can this file be made into a dot file so it can be hidden? Some systems (e.g. travis) support both hidden and non-hidden config files. If possible, I'd prefer hidden since this just clutters the root directory.

Reply:
Unfortunately the github action doesn't support the hidden file option and explicitly requires it be named that. Sonar-scanner cmd provides the project.settings option, which they do not. We could fork the repo to allow us to use this property, but that opens a host of maintenance issues we mioght not want to deal with.

Reply:
Definitelyl not worth forking, this is fine.